XCUITests/HomePageSettingsUITest.swift

```diff
@@ -8,6 +8,7 @@ let websiteUrl1 = "www.mozilla.org"
 let websiteUrl2 = "developer.mozilla.org"
 let invalidUrl = "1-2-3"
 let exampleUrl = "test-example.html"
+var urlList: [String] = ["www.reddit.com", "www.cnn.com", "www.cvs.com", "www.macys.com", "www.lanebryant.com", "www.linkedin.com", "www.target.com", "gap.com", "burberry.com", "www.walgreens.com", "www.amazon.com", "www.facebook.com", "www.twitter.com", "www.usc.edu"]
```

Comment:
Instead of going to each website we can use a pre-loaded Data Base with all those pages appearing in the Top Sites list. See for example this [test suite ](https://github.com/mozilla-mobile/firefox-ios/blob/master/XCUITests/ActivityStreamTest.swift#L14). For each test the corresponding data base is loaded with the data needed. It will be a good approach for this test, we have all the top sites there and depending on the number of rows selected, they are shown or not. 
The [data base for ipad](https://github.com/mozilla-mobile/firefox-ios/blob/master/XCUITests/ActivityStreamTest.swift#L17) could be used for iPhone, but I'm not sure if you need more top sites for the test on iPad sim (there are more top sites per row...)
These are the [existing DBs](https://github.com/mozilla-mobile/firefox-ios/tree/master/test-fixtures) in the repo. You can download and see the data of each one. There is also this [folder with more DBs](https://drive.google.com/drive/folders/1Nnb_q0kuw84wHgnFVTxn2eMcFVEi9BGT?usp=sharing) The number indicate the number of visited websites. If non of those is useful for you, you can create a new one following the steps described in the [wiki](https://github.com/mozilla-mobile/firefox-ios/wiki/Automated-UI-Tests#pre-loaded-data) Hope that helps! Let me know if you have questions/doubts

Reply:
With the DDBB implementation, this is not needed anymore or is it?